app/src/main/res/drawable/edit_text_border.xml

```diff
@@ -0,0 +1,11 @@
+<shape
```

Comment:
instead of adding this border add [textInputLayout](https://material.io/components/text-fields)